types_test.go

```diff
@@ -628,6 +628,84 @@ func TestIsHostname(t *testing.T) {
 	}
 }
 
+func TestIsGCPProjectId(t *testing.T) {
+	tests := []struct {
+		name   string
+		target string
+		want   bool
+	}{
+		{
+			name:   "GCP Project ID with length equal to or greater than 6 characters",
+			target: "rabbit",
+			want:   true,
+		},
+		{
+			name:   "GCP Project ID length should not less than 6 characters",
+			target: "hello",
+			want:   false,
+		},
+		{
+			name:   "GCP Project ID with length equal to or less than 30 characters",
+			target: "bagase-crucible-bubble-gorilla",
+			want:   true,
+		},
+		{
+			name:   "GCP Project ID length should be greater than 30 characters",
```

Comment:
`s/GCP Project ID length should be greater than 30 characters/GCP Project ID length should not be greater than 30 characters`?